plasmapy/particles/ionization_state.py

```diff
@@ -161,7 +161,7 @@ class IonizationState:
 
     Parameters
     ----------
-    particle: `particle_like`
+    particle: `ParticleLike`
```

Comment:
```suggestion
    particle: `~plasmapy.particles.particle_class.ParticleLike`
```

Otherwise the `intersphinx` linking does not work.